cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,25 @@
 /// <reference types="cypress" />
 
+beforeEach('My home page', () => {
+  cy.visit('https://conduit.mate.academy/user/login');
+});
+
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    cy.get('h1').should('contain.text', 'Sign in');
+
+    cy.get('[placeholder=Email]')
+      .type('denis_ser@gmail.com');
+
+    cy.get('[placeholder=Password]')
+      .type('tramadol228syadu');
+
+    cy.get('.btn')
+      .click();
+
+    cy.contains('a', 'denis_ser')
```

Comment:
here we duplicate username check (in contains() and should()), better not to do it twice, you can write it
cy.contains('a', 'denis_ser').should('be.visible')